services/scheduler/src/main/java/org/mule/service/scheduler/internal/config/ThreadPoolsConfig.java

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.service.scheduler.internal.config;
+
+import static java.io.File.separator;
+import static java.lang.Integer.parseInt;
+import static java.lang.Runtime.getRuntime;
+import static java.lang.System.getProperty;
+import static java.util.regex.Pattern.compile;
+import static org.slf4j.LoggerFactory.getLogger;
+
+import org.mule.runtime.api.exception.MuleException;
+import org.mule.runtime.core.api.DefaultMuleException;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+import java.util.regex.Pattern;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+
+import org.slf4j.Logger;
+
+/**
+ * Bean that contains the thread pools configuration for the runtime.
+ *
+ * @since 4.0
+ */
+public class ThreadPoolsConfig {
+
+  private static final Logger logger = getLogger(ThreadPoolsConfig.class);
+
+  public static final String PROP_PREFIX = "SchedulerService.";
+
+  private static final String NUMBER_OR_C_REGEXP = "([0-9]+(\\.[0-9]+)?)|(C)";
+  private static final Pattern POOLSIZE_PATTERN =
+      compile("(" + NUMBER_OR_C_REGEXP + ")?(\\s*[-+\\/*]\\s*(" + NUMBER_OR_C_REGEXP + ")?)*");
+
+  /**
+   * Loads the configuration from the {@code &#123;mule.home&#125;/conf/scheduler-pools.conf} file.
+   * 
+   * @return The loaded configuration, or the default if the file is unavailable.
+   * @throws MuleException for any trouble that happens while parsing the file.
+   */
+  public static ThreadPoolsConfig loadThreadPoolsConfig() throws MuleException {
+    File muleHome = getProperty("mule.home") != null ? new File(getProperty("mule.home")) : null;
```

Comment:
Is there an existing constant you can use for this, if not then introduce one here in this class.